poseidon2/io.nim

```diff
@@ -1,8 +1,9 @@
 import ./types
 import constantine/math/arithmetic
 import constantine/math/io/io_bigints
+import constantine/math/config/curves
 
-func unmarshal*(_: type F, bytes: openArray[byte]): F =
+func fromBytes*(_: type F, bytes: openArray[byte]): F =
```

Comment:
I think this should indicate the endianness of the expected bytes? Also, it's probably better to use an array, if the size is fixed?

Reply:
We no longer allow for big-endian bytes, it became too complicated.

An array parameter would imply more copying when used to convert in chunks, as happens in the function below.